openpype/settings/defaults/project_settings/houdini.json

```diff
@@ -14,48 +14,99 @@
+            "default_variants": ["master"]
+        },
         "CreateAlembicCamera": {
             "enabled": true,
-            "defaults": []
+            "default_variant": "Main",
+            "default_variants": ["Main"]
+        },
+        "CreateBGEO": {
+            "enabled": true,
+            "default_variant": "Main",
+            "default_variants": ["Main"]
         },
         "CreateCompositeSequence": {
             "enabled": true,
-            "defaults": []
+            "default_variant": "Main",
+            "default_variants": ["Main"]
+        },
+        "CreateHDA": {
+            "enabled": true,
+            "default_variant": "Main",
+            "default_variants": ["Main"]
+        },
+        "CreateKarmaROP": {
+            "enabled": true,
+            "default_variant": "master",
+            "default_variants": ["master"]
+        },
+        "CreateMantraROP": {
+            "enabled": true,
+            "default_variant": "master",
+            "default_variants": ["master"]
         },
         "CreatePointCache": {
             "enabled": true,
-            "defaults": []
+            "default_variant": "Main",
+            "default_variants": ["Main"]
+        },
+        "CreateRedshiftProxy": {
+            "enabled": true,
+            "default_variant": "master",
+            "default_variants": ["master"]
         },
         "CreateRedshiftROP": {
             "enabled": true,
-            "defaults": []
+            "default_variant": "Main",
+            "default_variants": ["Main"]
         },
         "CreateRemotePublish": {
             "enabled": true,
-            "defaults": []
+            "default_variant": "Main",
+            "default_variants": ["Main"]
         },
-        "CreateVDBCache": {
+        "CreateReview": {
             "enabled": true,
-            "defaults": []
+            "default_variant": "Main",
+            "default_variants": ["Main"]
         },
         "CreateUSD": {
             "enabled": false,
-            "defaults": []
+            "default_variant": "Main",
+            "default_variants": ["Main"]
         },
         "CreateUSDModel": {
             "enabled": false,
-            "defaults": []
+            "default_variant": "Main",
+            "default_variants": ["Main"]
         },
         "USDCreateShadingWorkspace": {
             "enabled": false,
-            "defaults": []
+            "default_variant": "Main",
+            "default_variants": ["Main"]
         },
         "CreateUSDRender": {
             "enabled": false,
-            "defaults": []
+            "default_variant": "Main",
+            "default_variants": ["Main"]
+        },
+        "CreateVDBCache": {
+            "enabled": true,
+            "default_variant": "Main",
+            "default_variants": ["Main"]
+        },
+        "CreateVrayROP": {
+            "enabled": true,
+            "default_variant": "master",
+            "default_variants": ["master"]
```

Comment:
Shouldn't this also be `Main`?

Reply:
yeah... I basically prioritized backwards compatibiliy here before they got deleted at https://github.com/ynput/OpenPype/pull/5367/files by @mustafa-zarkash . I personally will be overriding all the default subsets in our settings but happy to change to some other more reasonable defaults

Reply:
the default as milan mentioned is to stick with `"Main"`
https://github.com/ynput/OpenPype/pull/5367#pullrequestreview-1557577660